weni/components/__init__.py

```diff
@@ -0,0 +1,25 @@
+__all__ = [
+    "Component",
+    "Text",
+    "Header",
+    "Footer",
+    "Attachments",
+    "QuickReplies",
+    "ListMessage",
+    "CTAMessage",
+    "Location",
+    "OrderDetails",
+]
+
+from .component import (
+    Component,
+    Text,
+    Header,
+    Footer,
+    Attachments,
+    QuickReplies,
+    ListMessage,
+    CTAMessage,
+    Location,
+    OrderDetails,
+)
```

Comment:
Reverse the order between import and __all__ declaration

```suggestion
from .component import (
    Component,
    Text,
    Header,
    Footer,
    Attachments,
    QuickReplies,
    ListMessage,
    CTAMessage,
    Location,
    OrderDetails,
)


__all__ = [
    "Component",
    "Text",
    "Header",
    "Footer",
    "Attachments",
    "QuickReplies",
    "ListMessage",
    "CTAMessage",
    "Location",
    "OrderDetails",
]
```